wolfcrypt/src/rsa.c

```diff
@@ -1177,11 +1177,10 @@ static int RsaPad_PSS(const byte* input, word32 inputLen, byte* pkcsBlock,
     XMEMCPY(m, input, inputLen);
     m += inputLen;
     o = (int)(m - s);
-    if (saltLen > 0) {
```

Comment:
Can saltLen not be zero?
We don't want to make a call if not necessary.

Reply:
@SparkiDev, got it. saltLen can zero.
-1 <= saltLen <= hashLen
it is best to avoid the call. 
Closing the PR.